types/params.go

```diff
@@ -247,11 +251,11 @@ func (params ConsensusParams) ValidateBasic() error {
 		return fmt.Errorf("Feature.PbtsEnableHeight cannot be negative. Got: %d", params.Feature.PbtsEnableHeight)
 	}
 
+	// FIXME: should we enforce something more than > 0? Maybe >= time.Millisecond?
```

Comment:
I would be in favor of that, for practical reasons. @lasarojc ? @glnro ?

Reply:
I don't see the reason. If it is large, may be too large. And if it is small, like `time.Millisecond`, then it is just as good as checking `<= 0` imho.
Unless having an increment of less than 1ms could problematic, which I don't think it is since timestamps have ns resolution.

Reply:
OK, so let's keep it simple. Let's leave the `>0`

Reply:
Up to you.

Reply:
So remove the FIXME?